src/components/popup.tsx

```diff
@@ -72,6 +72,14 @@ export class Popup extends React.Component<PopupProps, PopupState> {
     this._secondaryBtnHandler = this._secondaryBtnHandler.bind(this);
   }
 
+  componentDidMount(): void {
+    const { inputDefaultValue } = this.props;
+
+    if (inputDefaultValue){
+      this.setState({inputValue: inputDefaultValue.toString()})
```

Comment:
Why are we copying the props to state?

Reply:
To set the default value of the input field. This is only happening on componentDidMount. I will add a comment